db/people.yml

```diff
@@ -5338,26 +5338,23 @@ swhan:
 swu:
   active: true
   aka: []
-  avatar: "https://github.com/\x16SeanWu25.png"
-  bio: Sean is a third year undergraduate student at Pepperdine University. He is
-    studying Computer Science/Mathematics and is interested in advancing medicine with machine learning.
-    He loves to play basketball and piano. He is conducting research with the Columbia SURE
-    program for Summer 2023.
-  education: [Pepperdine]
-  email: "\x16sean.wu@pepperdine.edu"
+  avatar: https://www.gravatar.com/avatar/00000000000000000000000000000000?d=mp&f=y
+  bio: 'Sean is a third year undergraduate student at Pepperdine University. He is studying Computer Science/Mathematics and is interested in advancing medicine with machine learning. He loves to play basketball and piano. He is conducting research with the Columbia SURE program for Summer 2023.'
+  education:
+    - begin_year: 2023
+      institution: pepperdine university
+      degree: visiting student
   employment:
-    - begin_date: 2023-05-31
-      end_date: 2023-08-04
-      organization: columbiau
-      position: undergraduate researcher
-      group: bg
+    - begin_year: 2023
+      end_year: 2023
+      end_month: 8
+      begin_month: 5
+      position: visiting student
+      organization: pepperdine university
       advisor: sbillinge
-      status: undergrad
-  github_id: "\x16SeanWu25"
```

Comment:
this is required, please add back, we just need to remove the rogue `\x16`

Reply:
On it right now